### PR TITLE
fix(config): add GE/Jasco/Enbrighten 14296

### LINKED ISSUE #2741: Missing device: GE/Jasco/Embrighten Z-Wave Plus Smart Dimmer Toggle 14296
**Which device is missing?**

GE/Jasco/Embrighten Z-Wave Plus Smart Dimmer Toggle
https://byjasco.com/ge-z-wave-plus-wall-smart-dimmer-light-almond-toggle-500s
https://www.amazon.com/Enbrighten-QuickFit-SimpleWire-Assistant-14296/dp/B08562L9RP

**What are the IDs?**

Manufacturer ID: 0x0063  
Product type: 0x4944 
Product ID: 0x3132 

**Did you check if we really don't have it?**

- [x] It is not in the [configuration DB](https://devices.zwave-js.io/) -- search for `14296` returns no results
- [x] It was not [merged recently](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Apr+is%3Aclosed) or has a [pending PR](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Aopen+is%3Apr)
- [x] It is not in the [TODO list](https://github.com/zwave-js/node-zwave-js/issues/1600)

**Do you have a link to the PDF manual?**

https://byjasco.com/mwdownloads/download/link/id/1567/

If so, it helps in creating the config file.

### PR DESCRIPTION
**Pull request by gregsheremeta**
add GE/Jasco/Enbrighten 14296, which is the same as 14295 but in light almond color

Fixes: #2741

### CODE REVIEW

**Comment by gregsheremeta** [comment]
before:

Selection_028

Selection_029

after:

Selection_031

Selection_030

File: packages/config/config/devices/0x0063/14296_zw3011.json [hunk 1]
```diff
@@ -0,0 +1,153 @@
+{
+	"manufacturer": "GE/Jasco",
+	"manufacturerId": "0x0063",
+	"label": "14296 / ZW3011",
+	"description": "In-Wall Smart Toggle Dimmer",
+	"devices": [
+		{
+			"productType": "0x4944",
+			"productId": "0x3132",
+			"zwaveAllianceId": 3352
```
Comment (gregsheremeta) [inline packages/config/config/devices/0x0063/14296_zw3011.json]: this product doesn't appear in zwave alliance, so I left the ID of the product I copied from, 14295

is this ok?

Comment (blhoward2) [reply packages/config/config/devices/0x0063/14296_zw3011.json]: No. Remove it if it's not in the zwa.

Comment (gregsheremeta) [reply packages/config/config/devices/0x0063/14296_zw3011.json]: done

**Review by blhoward2 — APPROVED** [review]

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**